src/__stories__/Documentation.stories.mdx

```diff
@@ -1,7 +1,6 @@
 import {
```

Comment:
@philibea I seem to remember that we can migrate these file to tsx since the last updates ?

Reply:
@chambo-e I think we said we keep pure documentation in MDX as it is easier to read & write and only stories of components in tsx

Reply:
Afaik MDX prevents us from fully using react 17 automated runtime, maybe it's supported now

Reply:
Alright so apparently storybook is now able to have the new react runtime in mdx so I'll close this :)